### PR TITLE
Tax Rate information update

### PR DESCRIPTION
**Pull request by Amy-Roberts**
Tax Rate information update

# Description
Tax Rate information update 

Include a summary of the new content you're adding or the reason for the change. 
Include relevant context.
Don't link to work items.

## Type of change

Please delete options that are not relevant.

- [ / ] New document(s)/updating existing
- [ ] Fixes
- [ ] Styling
- [ ] Bug fix (non-breaking change which fixes an issue)

## Reviews and merging

You are responsible for getting your PR merged. Address review comments promptly and **make sure to merge the PR when ready**.
Feel free to 'Enable automerge' - your PR will automatically merge when accepted and passing the build.

### CODE REVIEW

File: docs/sync-for-expenses/configandcategorize.md [hunk 1]
```diff
@@ -162,7 +162,9 @@ When pushing an expense reconciliation, you can include a tracking category to a
 
 The tax rates enable your SMB's to accurately track taxes against purchases and, depending on the locale, allow them to recoup the tax.
 
-Accounting systems typically store a set of taxes and associated rates within the accounting package. This means that users don't have to look up or remember the rates for each type of tax. For example: Applying the tax "UK sales VAT" to line items of an invoice adds the correct rate of 20%.
+Accounting systems typically store a set of taxes and associated rates within the accounting package. This means that users don't have to look up or remember the rates for each type of tax. For example: Applying the tax "UK sales VAT" to line items of an invoice adds the correct rate of 20%.
```
Comment (pzaichkina) [inline docs/sync-for-expenses/configandcategorize.md:165]:
```suggestion
Accounting systems typically store a set of taxes and associated rates within the accounting package. This means that users don't have to look up or remember the rates for each type of tax. For example, applying the tax "UK sales VAT" to line items of an invoice adds the correct rate of 20%. 
```

**Review by pzaichkina — CHANGES_REQUESTED** [review]
please address the comments

File: docs/sync-for-expenses/configandcategorize.md [hunk 2]
```diff
@@ -162,7 +162,9 @@ When pushing an expense reconciliation, you can include a tracking category to a
 
 The tax rates enable your SMB's to accurately track taxes against purchases and, depending on the locale, allow them to recoup the tax.
 
-Accounting systems typically store a set of taxes and associated rates within the accounting package. This means that users don't have to look up or remember the rates for each type of tax. For example: Applying the tax "UK sales VAT" to line items of an invoice adds the correct rate of 20%.
+Accounting systems typically store a set of taxes and associated rates within the accounting package. This means that users don't have to look up or remember the rates for each type of tax. For example, applying the tax "UK sales VAT" to line items of an invoice adds the correct rate of 20%.
```
Comment (d-laing) [inline docs/sync-for-expenses/configandcategorize.md:165]:
```suggestion
Accounting systems typically store a set of taxes and associated rates within the accounting package. This means users don't have to look up or remember the rate for each type of tax. For example, applying the tax "UK sales VAT" to the line items of an invoice adds the correct tax rate of 20%. 
```

File: docs/sync-for-expenses/configandcategorize.md [hunk 3]
```diff
@@ -162,7 +162,9 @@ When pushing an expense reconciliation, you can include a tracking category to a
 
 The tax rates enable your SMB's to accurately track taxes against purchases and, depending on the locale, allow them to recoup the tax.
 
-Accounting systems typically store a set of taxes and associated rates within the accounting package. This means that users don't have to look up or remember the rates for each type of tax. For example: Applying the tax "UK sales VAT" to line items of an invoice adds the correct rate of 20%.
+Accounting systems typically store a set of taxes and associated rates within the accounting package. This means that users don't have to look up or remember the rates for each type of tax. For example, applying the tax "UK sales VAT" to line items of an invoice adds the correct rate of 20%. 
+
+Assigning a tax rate to a transaction is mandatory unless the transaction is a [transfer](/sync-for-expenses/sync-process/expense-transactions#transaction-types). In some cases, your customers might not need to track tax on expenses. We recommend assigning a default tax code for 0% from the accounting package for these transactions.
```
Comment (d-laing) [inline docs/sync-for-expenses/configandcategorize.md:167]:
```suggestion
Assigning a tax rate to a transaction is mandatory unless the transaction is a transferIn or transferOut (see [Transaction types](/sync-for-expenses/sync-process/expense-transactions#transaction-types). In some cases, your customers might not need to track tax on expenses. We recommend assigning a default tax code for 0% from the accounting package for those transactions.
```

Comment (d-laing) [reply docs/sync-for-expenses/configandcategorize.md:167]: I don't think the link text of "transfer" is clear enough. I thought it was going to direct me to the data model.

**Review by d-laing — CHANGES_REQUESTED** [review]
Nice additions. I added review comments.

File: docs/sync-for-expenses/configandcategorize.md (hunk 3 above)
Comment (d-laing) [reply docs/sync-for-expenses/configandcategorize.md:167]: Use "those" to refers to things which have already been mentioned in the sentence :-) [1]

[1] https://www.collinsdictionary.com/dictionary/english/those

**Review by mcclowes — APPROVED** [review]

**Pull request merged**